lib/commands.coffee

```diff
@@ -62,15 +62,44 @@ module.exports =
 		# Default image to the given path
 		operation.from.image ?= image
 		operation.to.image ?= image
-
-		return imagefs.copy(normalizeDefinition(operation.from), normalizeDefinition(operation.to))
+		fromDefinition = normalizeDefinition(operation.from)
+		toDefinition = normalizeDefinition(operation.to)
+
+		return imagefs.interact(
+			fromDefinition.image
+			fromDefinition.partition
+			(_fs) ->
```

Comment:
Why the underscore ?

Reply:
@zvin because there is also fs as in require('fs') here. And I wanted to make it obvious that it's not exactly the same as fs.